File: src/test/java/com/zuhlke/interview/CodingChallengeTest.java
package com.zuhlke.interview;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class CodingChallengeTest {
    @Test
    public void testSayHello() {
        Assertions.assertEquals("Hello", new CodingChallenge().sayHello());
    }
}

File: src/test/kotlin/com/zuhlke/interview/CodingChallengeTest.kt
package com.zuhlke.interview

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class CodingChallengeTest {
    @Test
    fun testSayHello() {
        Assertions.assertEquals("Hello", CodingChallenge().sayHello())
    }
}
